README.md

```diff
@@ -35,38 +38,43 @@ Alert Logic Software Development Kit for Python allows developers to integrate w
 Launch python interpreter and then type:
 
 	```
-	>>> import almdrlib
-	>>> aims = almdrlib.client("aims")
-	>>> res = aims.get_account_details()
-	>>> print(f"{res.json()}")
+	import almdrlib
+	aims = almdrlib.client("aims")
+	res = aims.get_account_details()
+	print(f"{res.json()}")
 	```
 
 
 ## Development
 
 ### Getting Started
-> Prerequisites:
->
+
+####Prerequisites:
```

Comment:
I think you need a space between the # and the P. Some markdown parsers are picky, and won't make it a heading without the space.

Reply:
(addressed)